packages/input-option/src/InputOption/themes.ts

```diff
@@ -0,0 +1,169 @@
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+
+const State = {
+  Default: 'default',
+  Hover: 'hover',
+  Focus: 'focus',
```

Comment:
`highlight` since it may not be true keyboard focus in all cases

Reply:
Updated, you're right